### PR TITLE
KCAL-33: Include branch_extension documention for menus

### PR DESCRIPTION
**Pull request by harenson**
Related PR: https://github.com/2600hz/kazoo-callflow/pull/103
Depends on: https://github.com/2600hz/kazoo-core/pull/373

### CODE REVIEW

File: priv/api/swagger.json [hunk 1]
```diff
@@ -8377,7 +8377,7 @@
                     "type": "array"
                 },
                 "hunt": {
-                    "default": true,
+                    "default": false,
```
Comment (lazedo) [inline priv/api/swagger.json]: why is this changing ?

**Review by lazedo — CHANGES_REQUESTED** [review]

File: priv/api/swagger.json (hunk 1 above)
Comment (harenson) [reply priv/api/swagger.json]: Because when you create a new menu, that value is false by default.

Comment (lazedo) [reply priv/api/swagger.json]: and when did that changed without updating this ? afaics, its not related to this pr

Comment (harenson) [reply priv/api/swagger.json]: I just reverted that change.

**Comment by lazedo** [comment]
@harenson i think that `stop_after_extension` is not correct as the executing may not call `cf_exe:stop/1`, we should use `branch_extension` (or some rewording around that) that would reflect what we really do, ie, **branch** or **continue**

**Comment by harenson** [comment]
@lazedo Yes, makes sense.

**Pull request merged**